Acd 839 (#62)
Actindo-AG/Documentation: Core1Platform/HelpAndSupport/02_CreateTicket.md

> # Create ticket

Reviewer: H1 und H2 sollten nicht gleich heißen. Kann man hier einen allgemeineren Titel verwenden, z.B. Support function? Oder in H2 was anderes verwenden?

> The creation of a ticket is necessary, if something happens in your *Actindo Core1 Platform* that you think may be a bug in the Actindo software. In this case, the Actindo team has the chance to solve the bug and can provide an error correction to you and all other customers.   

Reviewer: ```suggestion
The creation of a ticket is necessary if something happens in your *Actindo Core1 Platform* that you think may be a bug in the Actindo software. In this case, the Actindo team has the chance to solve the bug and can provide an error correction to you and all other customers.   
```
--> Wollen wir wirklich das zugeben? ;-) Ich würde es anders/breiter formulieren, z.B.: if you are experiencing performance issues with your *Actindo Core1 Platform*... oder if you need support from our MPS team.

Author: you think may be a bug in the Actindo software

For example, one of the following situations may be a reason to create a ticket:   
- A workflow is failing
- The main account is not reachable
- Products show incorrect information on the frontend   

> > [INFO] Before creating a ticket, it may be useful to first check whether another users have not already reported the discovered issue. To do this, you can check the *Consulting* (MPS agreement (Managed Professional Services)) or the *Support* (older agreement) module.

Reviewer: INFO -> Info

> > [INFO] Before creating a ticket, it may be useful to first check whether another users have not already reported the discovered issue. To do this, you can check the *Consulting* (MPS agreement (Managed Professional Services)) or the *Support* (older agreement) module.

Reviewer: ```suggestion
> [Info] Before creating a ticket, it may be useful to first check whether other users have already reported the discovered issue. To do this, you can check the *Consulting* (MPS  (Managed Professional Services) agreement) or the *Support* (older agreement) module.
```

another users --> other users
https://dictionary.cambridge.org/de/grammatik/britisch-grammatik/other-others-the-other-or-another?q=another

INFO -> Info (durchgängig)

Author: d

## Create ticket

> Create a ticket for an issue that you have discovered. To speed up the support process we ask you to consider the instructions below when creating tickets.

Reviewer: encounter statt discover?

Author: discover

#### Prerequisites

- You have the permission to create tickets.

#### Procedure

*Actindo Core1 Platform > Any workspace*

![Core1 Platform](../../Assets/Screenshots/Core1Platform/UsingCore1/Core1Ticket.png "[Core1 Platform]")   

> 1. Click the ![Support](../../Assets/Icons/CreateTicket.png "[Support]") (Support) button.

Reviewer: ```suggestion
1. Click the ![Support](../../Assets/Icons/CreateTicket.png "[Support]") (Support) button.   
```

The *Contact us* window is displayed.

>    ![Contact us](../../Assets/Screenshots/Core1Platform/UsingCore1/Core1TicketTemplate.png "[Contact us")   

Reviewer: ```suggestion
   ![Contact us](../../Assets/Screenshots/Core1Platform/UsingCore1/Core1TicketTemplate.png "[Contact us]")   
```

2. Enter a descriptive reason in the *Reason* field. The reason description should be as precise as possible, for example "Delivery bills cannot be opened in the invoicing system".  An imprecise/bad example for the same fact would be "Invoicing does not work". 

> 3. Enter a detailed description of the issue. Include the following information:

Reviewer: 3. und 4. sollten eigentlich in einem Schritt sein, da sie sich zum selben Field beziehen.

     - Under which menu item did the issue occur?
     - What steps are needed to reproduce the behavior?
     - Since when does the issue occur and does it occur regularly?
    - How does the behavior deviate from the expected behavior?    

> 4. Enter a detailed description of the issue in the *Description* section. If the *Actindo Core1 platform* has output an error message, enter the exact wording of the error message. You can extend the description section by dropping the ![Extend](../../Assets/Icons/Extend.png "[Extend") (Extend) button downwards.   

Reviewer: ```suggestion
4. Enter a detailed description of the issue in the *Description* section. If the *Actindo Core1 platform* has output an error message, enter the exact wording of the error message. You can extend the description section by dropping the ![Extend](../../Assets/Icons/Extend.png "[Extend]") (Extend) button downwards.   
```

Author: done

   If there are issues with the communication with external systems such as shops, specify the software used including version (for example, Shopware 5.3.1) as well as the version of the connector in case of online shops.  
   For data protection reasons, do not write any unsolicited access data in the ticket.

5. If possible, take a screenshot of the error message or the screen on which the issue occurs.    
   Ideally, the issue is color-coded and the entire Actindo interface is visible. Make sure that no personal data is visible for data protection reasons.

> 5. If applicable, click the ![Attachment](../../Assets/Icons/Attachment.png "[Attachment") (Attachment) button to upload the screenshot or other documents. For security reasons, only file attachments in JPEG, PNG, PDF, and CSV format are allowed.   

Reviewer: ```suggestion
5. If applicable, click the ![Attachment](../../Assets/Icons/Attachment.png "[Attachment]") (Attachment) button to upload the screenshot or other documents. For security reasons, only file attachments in JPEG, PNG, PDF, and CSV format are allowed.   
```

Author: done

>    The file explorer of your device is opened. Select the file and upload it.

Author: nein, das gehört zusammen
